build.gradle

```diff
@@ -12,7 +12,7 @@ repositories {
 
 
 dependencies {
-	compile 'net.java.dev.jna:jna:3.5.2'
+	compile 'net.java.dev.jna:jna:4.1.0'
```

Comment:
Please don't change it in this pull request anyway. This pull request is about adding Linux ARM compatibility, and this change has nothing to do with that. You can open another pull request to change it instead and I'll accept it after testing that it was a 4.0.0 issue (I never really investigated it...)